tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pytest
+
+
+@pytest.fixture
+def config_defaults():
+    _dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I guess we can't use pytest-datadir as usual because of the need for some of these files to be at the root of the app?

Reply:
Maybe this was just me not using it properly, but I was not able to get pytest-datadir to use a data folder where I wanted it (granted, this was a while ago, so perhaps this is no longer true). I want one set of common data that I can use in both unit and functional tests, but from what I remember, that library did not allow me to put it there. So, I wrote this function a while back, and have been using it ever since.